src/main/java/com/prototype/playground/exceptions/GenericExceptionResponse.java

```diff
@@ -0,0 +1,92 @@
+package com.prototype.playground.exceptions;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class GenericExceptionResponse {
+
+    private final int status;
+    private final String httpStatus;
```

Comment:
Not sure if we really need `status` and `httpstatus` as separate entries. Saw in some stackoverflow once. Same feeling about having a debugMessage field and exception string.